packages/core/src/codewhispererChat/tools/executeBash.ts

```diff
@@ -207,22 +215,68 @@ export class ExecuteBash {
             const stdoutBuffer: string[] = []
             const stderrBuffer: string[] = []
 
-            let firstChunk = true
-            let firstStderrChunk = true
+            // Use a closure boolean value firstChunk and a function to get and set its value atomically
```

Comment:
This entire comment is likely to cause confusion. The only really purpose of this function in this codebase is to save a few lines of code. That's fine, but that is different than what this comment says.